### PR TITLE
Release notes for PrestaShop 8.0.2

### PR DESCRIPTION
**Pull request by kpodemski**
Release notes for PrestaShop 8.0.2

### CODE REVIEW

File: content/news/releases/prestashop-8-0-2-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- [HTML tags are not taken into accoung in some pages](https://github.com/PrestaShop/PrestaShop/issues/29940)
+- [An exception was thrown when import a file with non .csv extension and download the csv created file](https://github.com/PrestaShop/PrestaShop/issues/28761)
+- [The alternative currency symbol is not displayed in the product V2 & V1](https://github.com/PrestaShop/PrestaShop/issues/28430)
+
+### Front office
+
+- [Guest were detected as logged user](https://github.com/PrestaShop/PrestaShop/issues/30991)
+- [It was impossible to add a first address with error "Invalid postcode" in some circumstances](https://github.com/PrestaShop/PrestaShop/issues/30973)
+
+### Webservoces
```
Comment (MatShir) [inline content/news/releases/prestashop-8-0-2-maintenance-release.md]:
```suggestion
### Webservices
```

File: content/news/releases/prestashop-8-0-2-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- [An exception was displayed in all migrated page after deleting language used by the employee](https://github.com/PrestaShop/PrestaShop/issues/30091)
+- [Enabling new theme was failing due to routes caching](https://github.com/PrestaShop/PrestaShop/issues/30011)
+- [The popup for data import was not well escape](https://github.com/PrestaShop/PrestaShop/issues/29959)
+- [HTML tags are not taken into accoung in some pages](https://github.com/PrestaShop/PrestaShop/issues/29940)
+- [An exception was thrown when import a file with non .csv extension and download the csv created file](https://github.com/PrestaShop/PrestaShop/issues/28761)
+- [The alternative currency symbol is not displayed in the product V2 & V1](https://github.com/PrestaShop/PrestaShop/issues/28430)
+
+### Front office
+
+- [Guest were detected as logged user](https://github.com/PrestaShop/PrestaShop/issues/30991)
```
Comment (MatShir) [inline content/news/releases/prestashop-8-0-2-maintenance-release.md]:
```suggestion
- [Guests were detected as a logged-in user](https://github.com/PrestaShop/PrestaShop/issues/30991)
```

**Review by MatShir — CHANGES_REQUESTED** [review]

File: content/news/releases/prestashop-8-0-2-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+- "8.0.x"
+---
+
+The second patch version for PrestaShop 8 is now available to download!
+
+![8.0.2 is available!](/assets/images/2023/02/banner_8.0.2_1534x424.png)
+
+Version 8.0.2 is available! In this patch, you can find almost [20 bug fixes](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.0.2+label%3A%22Bug+fix%22+). As a patch version, 8.0.2 brings further goodness and stability to the 8.0 branch.
+
+There were several noteworthy issues addressed in the milestone 8.0.2 of PrestaShop. The upgrade mechanism had a problem where it couldn't check if the PHP session had started on some webservers, which was reported by many community members. Additionally, significant problems with web services were resolved, including the inability to create new customers. Another noteworthy fix was made for enabling themes that use Symfony routes for their modules, which was reported by a community member. Finally, there were several under-the-hood fixes and a new version of [Distribution API client](https://github.com/PrestaShop/ps_distributionapiclient/releases/tag/v1.0.3) released to enhance the module upgrade process.
```
Comment (coffrep) [inline content/news/releases/prestashop-8-0-2-maintenance-release.md]:
```suggestion
There were several noteworthy issues addressed in PrestaShop's 8.0.2 milestone. The upgrade mechanism had a problem where it couldn't check if the PHP session had started on some webservers, which was reported by many community members. Additionally, significant problems with web services were resolved, including the inability to create new customers. Another noteworthy fix was made for enabling themes that use Symfony routes for their modules, which was reported by a community member. Finally, there were several under-the-hood fixes and a new version of [Distribution API client](https://github.com/PrestaShop/ps_distributionapiclient/releases/tag/v1.0.3) released to enhance the module upgrade process.
```

**Review by MatShir — APPROVED** [review]

File: content/news/releases/prestashop-8-0-2-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- [Webservice was not working properly on PrestaShop 8.0 with PHP 8.1](https://github.com/PrestaShop/PrestaShop/issues/30165)
+- [It was not possible to create new customers with the webservices](https://github.com/PrestaShop/PrestaShop/issues/26568)
+
+### Translations
+
+- [Translation string for "new" in BO was not working](https://github.com/PrestaShop/PrestaShop/issues/30916)
+
+### Upgrade mechanism
+
+- [With the 1-Click Upgrade module v4.15.0, It was not possible to create a PHP session and no new version was available](https://github.com/PrestaShop/PrestaShop/issues/30658)
```
Comment (coffrep) [inline content/news/releases/prestashop-8-0-2-maintenance-release.md]:
```suggestion
- [With the 1-Click Upgrade module v4.15.0, it was not possible to create a PHP session and no new version was available](https://github.com/PrestaShop/PrestaShop/issues/30658)
```

File: content/news/releases/prestashop-8-0-2-maintenance-release.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+### Other
+
+- [There was a bad PHP float comparison in Tools](https://github.com/PrestaShop/PrestaShop/issues/29652)
+
+## Acknowledgments
+
+Contributors to this patch version, from both the project members and the community at large:
+
+Alexandre Hellin, Boubker Bribri, Daniel Hlavacek, Ibrahima Sow, Krystian Podemski, Léa Delin, Mathieu Ferment, Matthieu Rolland, Maxime Flasquin, Nesrine Abdmouleh, Nicolas Lœuillet, Thomas Baccelli
```
Comment (coffrep) [inline content/news/releases/prestashop-8-0-2-maintenance-release.md]:
```suggestion
Alexandre Hellin, Boubker Bribri, Daniel Hlavacek, Ibrahima Sow, Krystian Podemski, Léa Delin, Mathieu Ferment, Matthieu Rolland, Maxime Flasquin, Nesrine Abdmouleh, Nicolas Lœuillet, Thomas Baccelli.
```

File: content/news/releases/prestashop-8-0-2-maintenance-release.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+There were several noteworthy issues addressed in the milestone 8.0.2 of PrestaShop. The upgrade mechanism had a problem where it couldn't check if the PHP session had started on some webservers, which was reported by many community members. Additionally, significant problems with web services were resolved, including the inability to create new customers. Another noteworthy fix was made for enabling themes that use Symfony routes for their modules, which was reported by a community member. Finally, there were several under-the-hood fixes and a new version of [Distribution API client](https://github.com/PrestaShop/ps_distributionapiclient/releases/tag/v1.0.3) released to enhance the module upgrade process.
+
+It is recommended to upgrade your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before. You can upgrade to the latest version using 1-Click Upgrade [v4.15.0](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.15.0).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-0-2-maintenance-release.md]:
```suggestion
It is recommended to upgrade your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before. You can upgrade to the latest version using the 1-Click Upgrade [v4.15.0](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.15.0).
```

**Review by coffrep — DISMISSED** [review]
Just a few suggestions

**Review by coffrep — APPROVED** [review]

**Pull request merged**